cms/static/js/spec/views/edit_chapter_spec.js

```diff
@@ -0,0 +1,27 @@
+define(['jquery', 'underscore', 'js/views/edit_chapter', 'js/models/course'],
+    function ($, _, EditChapter, Course) {
+    'use strict';
+
+        describe('EditChapter', function () {
+
+                beforeEach(function() {
+                    window.course = new Course({
+                        id: '1',
+                        name: '&amp;lt;Vedran&#39;s course&amp;gt;',
+                        url_name: 'course_name',
+                        org: 'course_org',
+                        num: 'course_num',
+                        revision: 'course_rev'
+                    });
+                });
+
+            describe('openUploadDialog', function() {
+
+                it('displays the encoded name of the course', function () {
+                    var title = _.template(('Upload a new PDF to "<%= name %>"'),
```

Comment:
Instead of duplicating the code you are testing, the idea is to execute the code being tested. You want a test point that fails without your bug fix in edit_chapter.js.